src/components/CommentCard/CommentCard.jsx

```diff
@@ -1,16 +1,30 @@
 import React from 'react'
 import './CommentCard.css'
 import moment from 'moment'
+import { deleteArticleComment } from '../../../APIs'
+
+
+export const CommentCard = ({ comment, user, setPostIndicator, postIndicator }) => {
+  const deleteComment = () => {
+
+    deleteArticleComment(comment.comment_id)
+      .then(() => {
+      setPostIndicator(++postIndicator)
+      })
+      .catch(()=>{
+        alert('The comment no longer exists')}
```

Comment:
What I'm about to say is very general advice applicable throughout your whole app with a sizeable chunk of opinion (using alerts is fine especially whilst you work on your core functionality), but you may want to consider alternative ways to feedback errors to users than relying on alerts. From a UX perspective they aren't the most accessible out of the box/a lot of users have pop up blockers nowadays so probably wouldn't receive them. 

Hint: conditional rendering/formatting 👀